### ISSUE TITLE
Disconnect?

### ISSUE THREAD

**Issue by JPRuehmann** [Johni0702/mumble-web#150]
Hello
That is exactly what I am looking for since Years.
How can I disconnect?
I can´t find a Button for that.
Thanks,

**Comment by Hyask** [comment]
Indeed, an explicit `Disconnect` button would be great, especially when Mumble-web is integrated as a Matrix widget in Element.
